README.md

```diff
@@ -422,6 +422,153 @@ Now you can do that by passing the `--wait-for-host` flag. The value of this opt
 - `--wait-for-no-redirect`               Do not follow HTTP 3xx redirects.
 - `--wait-for-timeout`                   Sets the timeout for all wait operations, 0 is unlimited. (default 10s)
 
+## (EXPERIMENTAL) Quantitative testing
+
+In the latest version of `go-ftw`, we have added a new feature that allows you to run quantitative tests.
+This feature is still experimental and may change in the future.
+
+### What is the idea behind quantitative tests?
+
+Quantitative tests allow you to run tests using payloads to quantify the amount of false positives you might get when running in production.
+We use a well-known corpora of text to generate payloads that are sent to the WAF. The WAF should not block these payloads, as they are not malicious.
+
+Anyone can create their own corpora of text and use it to test their WAF. The corpora of text is a list of strings that are sent to the WAF to check if it blocks them.
+
+The result of this test is a percentage of false positives. The lower the percentage, the better the WAF is at not blocking benign payloads.
```

Comment:
While I wrote the suggestion below, I wondered whether we might want to label "false positive rate" differently for default corpora. For example "generic FP rate", to make it clear that we cannot predict the behaviour of a rule in a specific context. What do you think?

```suggestion
The result of a test run is a percentage of false positives. The lower the percentage, the better the WAF is at not blocking benign payloads for a given rule. However, since we use generic corpora in our tests, the strings in those corpora will not necessarily be representative of the domain of a specific site. This means that a rule with a low false positive rate can still produce many false positives in specific contexts, e.g., when a website contains programming language code.
```

Reply:
Agreed to the text added.